<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-0EHH1C9" failures="1" tests="6" name="My Demo test" time="0.082" errors="0" timestamp="2021-11-03T12:37:42 IST">
  <testcase classname="com.qa.test.DemoTest" name="mul" time="0.012"/>
  <testcase classname="com.qa.test.DemoTest" name="sub" time="0.003">
    <failure type="java.lang.AssertionError" message="expected [10] but found [22]">
      <![CDATA[java.lang.AssertionError: expected [10] but found [22]
at com.qa.test.DemoTest.sub(DemoTest.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1541)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- sub -->
  <testcase classname="com.qa.test.DemoTest" name="sum" time="0.002"/>
  <testcase classname="com.qa.test.DemoTest2" name="mul" time="0.002"/>
  <testcase classname="com.qa.test.DemoTest2" name="sub" time="0.002"/>
  <testcase classname="com.qa.test.DemoTest2" name="sum" time="0.002"/>
</testsuite> <!-- My Demo test -->
